src/NewDataList/NewDataList.js

```diff
@@ -124,5 +124,19 @@ module.exports = kind({
 			Spotlight.spot(fv);
 			return true;
 		}
+	},
+	/**
+	* Called by moonstone/Scrollable.filterFocus()
+	* @private
+	*/
+	isVisibleChild: function(event){
```

Comment:
For consistency with `eventIsFromScrollingChild()`, and to match the nature of the argument, I'd recommend that we name this `eventIsFromVisibleChild()`.

@germboy, rather than kick it back again, I think you can go ahead and make this change as part of integrating.